shared-bindings/_bleio/Adapter.c

```diff
@@ -145,17 +145,19 @@ const mp_obj_property_t bleio_adapter_name_obj = {
 //|         :param buf data: advertising data packet bytes
 //|         :param buf scan_response: scan response data packet bytes. ``None`` if no scan response is needed.
 //|         :param bool connectable:  If `True` then other devices are allowed to connect to this peripheral.
+//|         :param bool anonymous:  If `True` then this device's MAC address is randomized regularly.
```

Comment:
We should stop advertising when the address rotates. Keeping the advertising data static while rotating the mac address removes the anonymization aspect of it.